lib/yt/models/asset.rb

```diff
@@ -68,7 +69,7 @@ def metadata_effective
       #   with the asset. You can apply a label to multiple assets to group
       #   them. You can use the labels as search filters to perform bulk updates,
       #   to download reports, or to filter YouTube Analytics.
-      has_attribute :label
+      has_attribute :label, default: []
```

Comment:
Does this change anything? Do we care?

Reply:
This won't change anything but I'm just following the same pattern used elsewhere in YT for attributes that return an array of data. Should I remove and let nil be the default?

Reply:
I agree with @jcohenho